modules/data.land/R/pool_ic_list2netcdf.R

```diff
@@ -0,0 +1,98 @@
+##' @name pool_ic_list2netcdf
+##' @title pool_ic_list2netcdf
+##' @export
+##'
```

Comment:
I think `#' @description` works, but am not sure because you can skip the tag entirely -- Roxygen treats untagged lines as description:

```
#' do a thing
#'
#' Takes input and turns it into output, as descibed blah blah blah...
#' blah blah blah...
#'
#' @export
#' @param x ...
#' ...
do_thing <- function(...){...
```